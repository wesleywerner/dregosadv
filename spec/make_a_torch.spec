brief
gonear bungalow portal
search rubble
go north
go north
search jungle
x torch
tie cloth to stick
x torch
inventory
gonear waterf portal
search waterfall
look in alcove
enter alcove
take stone
light torch with stone
enter temple
open urn
put torch in urn
light torch with stone
go out
