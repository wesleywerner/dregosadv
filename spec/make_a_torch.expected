  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Bungalow
  
  You can see some rubble here.
  An unexplored path is to the southwest. An unexplored path is to the north.
. 
> >  You scrounge amid the rubble and disturb a giant centipede - The critter
  scurries out of sight. You continue your search and find a strip of canvas.
  You take it.
  
> >  Clearing
  A hut is due south. An unexplored path is to the east.
  
> >  Lost in the Jungle
. 
> >  You find a stick.
  
> >  You can't see any such thing.
  
> >  You wrap the strip of canvas around the stick to make a crude torch.
  
> >  A wooden stick about the length of your arm.
  A crude wick is tied to the one end.
  
> >  You're carrying:
    a wooden torch
    your satchel (being worn)
      a matchbook
      your journal
      your compass
    your leather jacket (being worn)
    your fedora (being worn)
    your bullwhip (being worn)
  
> >  Waterfall
  
  A metallic blue glint catches your eye.
  A temple is due south. A rope bridge is due west.
  
> >  You lean into the narrow space behind the cascading water and see an alcove
  behind the waterfall.
  
  [You scored your first point! Type SCORE/FULL to list them.]
  
> >  It is pitch black in there. If only your torch was lit.
  
> >  It is pitch black in there. If only your torch was lit.
  
> >  You pick up the stone, it is a piece of flint.
  
> >  The wooden torch fails to burn -- The wick needs some kind of fuel.
  
> >  [Purloined.]
  
> >  You open the urn, revealing some oil.
  
> >  You put the wooden torch in the urn, it is now covered in oil.
  
  [You scored another point.]
  
> >  You strike the flint and sparks fly onto the wick, they catch and a small
  flame appears -- gradually the flame grows until it covers the wick.
  
> >  You see the passage beyond the alcove leading underground.
  
> >  You steady your torch and head down the dark passage.
  
  Underground Tomb
  
  A stone column stands in the center of the room.
  Some stairs leads upward.
  
> >